frb_codegen/src/utils.rs

```diff
@@ -73,6 +139,13 @@ pub fn get_symbols_if_no_duplicates(configs: &[crate::Opts]) -> Result<Vec<Strin
         ));
     }
 
+    if explicit_raw_symbols
+        .iter()
+        .any(|s| DART_KEYWORDS.contains(&s.as_str()))
+    {
+        return Err(anyhow!("api name cannot be a dart keyword"));
```

Comment:
uber-nit: would be great to add a bit more information in the message (e.g. which keyword violates it?)